File: 프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.py
def solution(my_string, num1, num2):
    my_list = list(my_string)  # 문자열을 리스트로 변환

    # num1과 num2의 위치에 있는 문자를 교환
    my_list[num1], my_list[num2] = my_list[num2], my_list[num1]

    return "".join(my_list)  # 리스트를 다시 문자열로 변환하여 반환

# 테스트 예제 실행
print(solution("hello", 1, 2))          # "hlelo"
print(solution("I love you", 3, 6))     # "I l veoyou"

File: 프로그래머스/0/120895. 인덱스 바꾸기/인덱스 바꾸기.java
class Solution {
    public String solution(String myString, int num1, int num2) {
        // 문자열을 문자 배열로 변환
        char[] charArray = myString.toCharArray();

        // num1과 num2의 위치에 있는 문자 교환
        char temp = charArray[num1];
        charArray[num1] = charArray[num2];
        charArray[num2] = temp;

        // 문자 배열을 다시 문자열로 변환하여 반환
        return new String(charArray);
    }

    public static void main(String[] args) {
        Solution sol = new Solution();

        // 테스트 예제 실행
        System.out.println(sol.solution("hello", 1, 2)); // "hlelo"
        System.out.println(sol.solution("I love you", 3, 6)); // "I l veoyou"
    }
}